params/hooks_libevm.go

```diff
@@ -74,7 +75,7 @@ func (r RulesExtra) ActivePrecompiles(existing []common.Address) []common.Addres
 	}
 
 	var addresses []common.Address
-	addresses = append(addresses, maps.Keys(precompiles)...)
+	addresses = slices.AppendSeq(addresses, maps.Keys(precompiles))
```

Comment:
(musing) This suggests that variadic expansion with `...` can be applied to iterators (at least in `append`), which is nice.

Reply:
Well TIL `musing` 🧠 🧠 

Out of the vocabulary realm, I don't think you can right? For example `append(addresses, maps.Keys(precompiles)...)` doesn't work.

Reply:
It looked like that was the exact pattern in the old code though, no?

Reply:
It was but the exp-version maps behaves differently than the std maps package